evcouplings/couplings/pairs.py

```diff
@@ -91,37 +87,49 @@ def enrichment(ecs, num_pairs=1.0, score="cn", min_seqdist=6):
         Sorted table with enrichment values for each
         position in the sequence
     """
-    # determine how many positions ECs are over
-    pos = set(ecs.i.unique()) | set(ecs.j.unique())
-    num_pos = len(pos)
+
+    # check if the provided table has segments...
+    if "segment_i" in ecs.columns and "segment_j" in ecs.columns:
+        has_segments = True
+    # ... and if not, create them
+    else:
+        has_segments = False
+        ecs["segment_i"] = "A_1"
```

Comment:
1. Please put default segment in some central location, I think we might already have one somewhere in couplings/mapping-land...

2. I never know when it is okay to leave away loc in assignments, so I would put ``ecs.loc[:, "segment_i"] = ...`` to be on the safe side

Reply:
I think the closest thing we had was in compare protocol line 832, 
```
    first_segment_name = Segment.from_list(kwargs["segments"][0]).segment_id
    second_segment_name = Segment.from_list(kwargs["segments"][1]).segment_id
```
Unless it's somewhere else and I can't find it

Reply:
Created it in couplings/mapping